testcases/testcase_blood_flow_model.py

```diff
@@ -14,53 +14,52 @@
-                             # 2: PyAMG solver
-                             # 3-...: other solvers
+        "read_network_option": 2,   # 1: generate hexagonal graph
+                                    # 2: import graph from csv files
+                                    # 3: import graph from igraph file (pickle file)
+                                    # 4: todo import graph from edge_data and vertex_data pickle files
+        "write_network_option": 1,      # 1: do not write anything
+                                        # 2: write to igraph format # todo: handle overwriting data from import file
+                                        # 3-...: todo other file formats.
+        "tube_haematocrit_option": 2,   # 1: No RBCs (ht=0)
+                                        # 2: Constant haematocrit
+                                        # 3: todo: RBC tracking
+                                        # 4-...: todo: steady state RBC laws
+        "rbc_impact_option": 3,     # 1: No RBCs (hd=0)
+                                    # 2: Laws by Pries, Neuhaus, Gaehtgens (1992)
+                                    # 3: Laws by Pries and Secomb (2005)
+                                    # 4-...: todo: Other laws. in vivo?
+        "solver_option": 1,     # 1: Direct solver
+                                # 2: PyAMG solver
+                                # 3-...: other solvers
 
         # Blood properties
         "ht_constant": 0.3,  # only required if RBC impact is considered
         "mu_plasma": 0.0012,
 
         # Hexagonal network properties. Only required for "read_network_option" 1
-        "nr_of_hexagon_x": 3,
-        "nr_of_hexagon_y": 3,
+        "nr_of_hexagon_x": 7,
+        "nr_of_hexagon_y": 7,
         "hexa_edge_length": 62.e-6,
         "hexa_diameter": 4.e-6,
         "hexa_boundary_vertices": [0, 27],
-        "hexa_boundary_values": [2, 1],
+        "hexa_boundary_values": [5, 2],
         "hexa_boundary_types": [1, 1],
 
         # Import network from csv options. Only required for "read_network_option" 2
-        "csv_path_vertex_data": "data/network/b6_B_pre_061/node_data.csv",
-        "csv_path_edge_data": "data/network/b6_B_pre_061/edge_data.csv",
-        "csv_path_boundary_data": "data/network/b6_B_pre_061/boundary_node_data.csv",
+        "csv_path_vertex_data": "/Users/cucciolo/Desktop/microBlooM/MVN2/node_data.csv",
+        "csv_path_edge_data": "/Users/cucciolo/Desktop/microBlooM/MVN2/edge_data.csv",
+        "csv_path_boundary_data": "/Users/cucciolo/Desktop/microBlooM/MVN2/node_boundary_data.csv",
         "csv_diameter": "D", "csv_length": "L",
         "csv_edgelist_v1": "n1", "csv_edgelist_v2": "n2",
         "csv_coord_x": "x", "csv_coord_y": "y", "csv_coord_z": "z",
-        "csv_boundary_vs": "nodeId", "csv_boundary_type": "boundaryType", "csv_boundary_value": "boundaryValue",
+        "csv_boundary_vs": "nodeID", "csv_boundary_type": "boundaryType", "csv_boundary_value": "p",
```

Comment:
Same here! Can you please leave the "default" options here?
"csv_boundary_vs": "nodeId", "csv_boundary_type": "boundaryType", "csv_boundary_value": "boundaryValue"